tools/individual-packages-gen/.gitignore

```diff
@@ -0,0 +1,15 @@
+*.js
```

Comment:
Not sure what you mean by this comment - what is not `.gitignore`d right now that should be...?

The contents of this file are dictated by `pkglint`.

Reply:
Does it make sense to use `cdk-build` (assume `pkglint` is brought in because of this) for this package?

Usually, `cdk-build` is intended to be used with our L2 packages where these rules are standardized and must not deviate.
I can see the case for loosening this requirement for private build tooling packages, and carry its own build scripts. Some of them in the `tools/` folder already do.

The only useful bit I can think of is the eslint rules that come with `cdk-build` but we can configure that explicitly.

Something to consider to see if that simplifies, esp around the `.gitignore`.

(Non-blocking)